scripts/ccg/align_tags_to_bert.py

```diff
@@ -0,0 +1,99 @@
+import sys
```

Comment:
Haven't verified the logic in detail. If you haven't already, run a few sanity checks to make sure we're not getting off-by-ones in position or index anywhere.

Reply:
Test written